controller.sh

```diff
@@ -26,13 +26,20 @@ case "$1" in
       -p ${DOCKER_STUN_PORT}:3478/udp \
       -p ${DOCKER_AIRCONTROL_PORT}:10001/udp \
       -e TZ='America/New_York' \
-      -v $(pwd)/testdata/unifi:/unifi \
       --name unifi \
       jacobalberty/unifi:${2:-$default_tag}
 
     echo "Waiting for login page..."
-    timeout 300 bash -c 'while [[ "$(curl --insecure -s -o /dev/null -w "%{http_code}" '"https://localhost:${DOCKER_HTTPS_PORT}/manage/account/login"')" != "200" ]]; do sleep 5; done'
+    timeout 300 bash -c 'while [[ "$(curl --insecure --location -s -o /dev/null -w "%{http_code}" '"https://localhost:${DOCKER_HTTPS_PORT}/manage/account/login"')" != "200" ]]; do sleep 5; done'
     echo "Controller running."
+
+    echo "Bootstrapping controller..."
+    function unifi_curl() {
+      curl --data "${2}" --header 'Content-Type application/json' --insecure "https://localhost:${DOCKER_HTTPS_PORT}${1}"
+    }
+    unifi_curl /api/cmd/sitemgr '{"cmd":"add-default-admin","name":"tfacctest","email":"tfacctest@example.com","x_password":"tfacctest1234"}'
```

Comment:
Not a big deal, but considering the repetition of user/pass, etc in this file, should they be vars?

Reply:
I'm going to make some other changes to this script soon so I'll just hold off for now.